CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.0.0
+### Changed
+- Regenerate SWIG wrapper code against TagLib 2.0.1. This breaks
+  compatibility with TagLib 1.x.
```

Comment:
1. Are there any high-level breaking changes worth mentioning? Maybe the things that we had to change in our tests?: https://github.com/robinst/taglib-ruby/pull/143/files#diff-5228a01ff614b6ec026b07c1bb623f7d8339b0c398102138bc59c50bec8922d3

2. Let's also add something along the lines of "For TagLib 1, please keep using taglib-ruby 1.x versions. There might still be new 1.x versions in the future if necessary, but development of new features will focus on 2.x."

3. Could we add some text about how to find out which major version of TagLib is installed on the system? E.g. on Mac, look at `TAGLIB_MAJOR_VERSION` in `/opt/homebrew/include/taglib/taglib.h`, on Linux, look there, etc.

Reply:
1. I can think of the `id3v2_tag` thing, I'll look if there are others
2. OK.
3. I don't see how that does not get vague or very wordy. It depends on the package manager where the header got installed. This (the changelog) is not the place to tell people how to use their package managers.

Reply:
Yeah, maybe the readme is a better place for it.

Reply:
FWIW, I wonder if this is a reasonably universal way to determine the installed TagLib version:

```
$ pkg-config --modversion taglib
2.0.1
```

At least it's distro-agnostic.

Reply:
Ah yeah, works on Mac with brew:

```
$ pkg-config --modversion taglib
1.13.1
```